Packaging/Sources/5_build_libs-gui.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+. /Developer/Makefiles/GNUstep.sh
+
+cd ./nextspace-gui-0.28.0+nextspace || exit 1
+
+make clean
+sh ./configure \
+	--disable-icu-config
```

Comment:
Why ICU library is disabled here? Do you have problems with it?

Reply:
hmm, I just copied this from the Debian.
OK, I'll double check!